2.6/run-mongod.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# For SCL enablement
+source /var/lib/mongodb/common.sh
+
+set -eu
+
+# Data directory where MongoDB database files live. The data subdirectory is here
+# because .bashrc and mongodb.conf both live in /var/lib/mongodb/ and we don't want a
+# volume to override it.
+export MONGODB_DATADIR=/var/lib/mongodb/data
+
+# Configuration settings.
+export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
+export MONGODB_SMALLFILES=${MONGODB_SMALLFILES:-true}
+export MONGODB_QUIET=${MONGODB_QUIET:-true}
+
+
+export MONGODB_KEYFILE_SOURCE_PATH="/var/run/secrets/mongo/keyfile"
+export MONGODB_KEYFILE_PATH="/var/lib/mongodb/keyfile"
+
+function usage() {
+  echo "You must specify following environment variables:"
+  echo "  MONGODB_USER"
+  echo "  MONGODB_PASSWORD"
+  echo "  MONGODB_DATABASE"
+  echo "  MONGODB_ADMIN_PASSWORD"
+  echo "Optional variables:"
+  echo "  MONGODB_REPLICA_NAME"
+  echo "MongoDB settings:"
+  echo "  MONGODB_NOPREALLOC (default: true)"
+  echo "  MONGODB_SMALLFILES (default: true)"
+  echo "  MONGODB_QUIET (default: false)"
```

Comment:
This is copy-pasting a mistake from 2.4, [line 16](https://github.com/openshift/mongodb/pull/60/files#diff-4dc45b8cc3069c3d1b885dd850849a39R16) makes `MONGODB_QUIET` default to `true`...